src/routes/auth.js

```diff
@@ -41,6 +41,18 @@ router.post("/register", async (req, res) => {
             "reason": "No username specified",
             "error_code": ERROR_TYPES.MISSING_FIELD,
         });
+    } else if (req.body.username.length > 20) {
+        return res.status(500).json({
+            "success": false,
+            "reason": "Username is too long",
+            "error_code": ERROR_TYPES.MISSING_FIELD,
+        });
+    } else if (req.body.username.length < 3) {
```

Comment:
Also, please remove the `else`s if each if already returns :sweat_smile: 

It's functionally the same but results in reduced visual clutter in my opinion :slightly_smiling_face: